src/redux/features/posts/postSlice.js

```diff
@@ -0,0 +1,82 @@
+import {createAsyncThunk, createSlice} from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const initialState={
+    createPost: [],
+    id:"",
+    postInput:"",
+    bookmarkPost:[],
+    commentInput:"",
+    editCommentInput:"",
+    commentId:"",
+    status:"idel",
+    
+}
+
+
+// export const loadPosts=createAsyncThunk("posts/loadPosts",async()=>{
+//     const { data } = await axios.get("/api/posts");
+//     return data.posts
+// })
```

Comment:
you can remove commented code if not required

Reply:
ok